samples/c/common/opencv_c_wrapper/src/bmp_reader.c

```diff
@@ -67,7 +67,7 @@ int readBmpImage(const char* fileName, BitMap* image) {
         CLEANUP_AND_RETURN(2);
     }
 
-    image->width = (image->infoHeader.width > 0) ? image->infoHeader.width : 0;
+    image->width = abs(image->infoHeader.width);
```

Comment:
Why is clipping replaced with `abs`?

Reply:
I think miss to revert it, as I've got success on coverity.
In fact the abs or clip should not be applied here as it bmp width should be non-negative (correct me if is not true), but height can be negative (rows order).

If I will got positive result on coverity (need time for get scan result) will revert it as it was, but proper solution would be limit bmp dimension to some range.